app/controllers/admin/api/services/proxies_controller.rb

```diff
@@ -74,6 +74,27 @@ def update
     respond_with(proxy)
   end
 
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/services/{service_id}/proxy/deploy.xml"
+  ##~ e.responseClass = "proxy"
+  #
+  ##~ op            = e.operations.add
+  ##~ op.httpMethod = "POST"
+  ##~ op.summary    = "Proxy Deploy"
+  ##~ op.description = "This will create a new APIcast configuration version for the Staging (and Production in case of Service Mesh) environment with the updated settings."
```

Comment:
```suggestion
  ##~ op.description = "Promotes the APIcast configuration to the Staging Environment (Production Environment in case of Service Mesh)."
```